Frameworks/Foundation/NSOrderedSet.mm

```diff
@@ -234,98 +315,104 @@ - (BOOL)containsObject:(id)object {
 - (void)enumerateObjectsAtIndexes:(NSIndexSet*)indexSet
                           options:(NSEnumerationOptions)opts
                        usingBlock:(void (^)(id, NSUInteger, BOOL*))block {
+    // Uncomment this when enumerateObjectsAtIndexes is implemented.
+    // if (block == nil) {
+    //  [NSException raise:NSInvalidArgumentException format:@"block can not be nil."];
+    //}
+    //[_arrayContainer enumerateObjectsAtIndexes:indexSet options:opts usingBlock:block];
```

Comment:
Please remove commented out code. In general commented out code just clutters the code base and makes things harder to read. Because commented out code by definition never compiles, it often is quickest to get stale and lacks the appropriate context for why the code isn't working, what needs done to enable it, etc.